data.sql

```diff
@@ -37,17 +37,18 @@ CREATE TABLE ResponseLog (
 -- Responders verify when they have arrived to help another user
 CREATE TABLE ArrivalLog (
     ID serial PRIMARY KEY,
-    ResponseID Integer NOT NULL,
+    ResponseID Integer NOT NULL UNIQUE, -- users arrive for an alert only once

```

Comment:
Can you remind me what the reason for this was? Or briefly describe a situation where this UNIQUE attribute is significant?

Reply:
It makes sure each responseLog has either 1 arrivalLog (responder made it to help) or 0 (responder didn't show up). If for some reason the backend gets called to make multiple arrivalLogs it won't add duplicate data

Reply:
so the first arrival status update will create a new log entry but no other one will be created after?

Reply:
Yes